README.md

```diff
@@ -288,6 +288,10 @@ try {
 // Current user
 $user = ParseUser::getCurrentUser();
 ```
+#### Session Id and Session Fixation
```

Comment:
Text looks good 👍 . Should probably add a link to this above 'Verification Emails' in the table of contents and we should be good.

Reply:
@acinader Can we get this in?

Reply:
i think I have just added what you're asking. :).

Reply:
I think he was referring Table of Contents in the ReadME

Reply:
riiight.  thanks for spoon feeding me.